lib/rules/no-unused-class.js

```diff
@@ -107,14 +109,14 @@ export default {
            ```
            then the loop will be run 2 times
          */
-        _.forOwn(map, (o) => {
+        _.forIn(map, (o) => {
```

Comment:
`map` is an object that you created as `{}`, which is a plain object in which it's prototype is Object.prototype. Unless someone modifies Object.prototype (which nobody does because it's dangerous), there's no need to use `forOwn`.

`forOwn` is basically a `forIn` with a `hasOwnProperty` check in each iteration.